topologytest/hlv_test.go

```diff
@@ -138,7 +138,11 @@ func deleteConflictDocs(t *testing.T, dsName base.ScopeAndCollectionName, peers
 			continue
 		}
 		deleteVersion := peer.DeleteDocument(dsName, docID)
-		t.Logf("deleteVersion: %+v", deleteVersion)
+		if peer.Type() == PeerTypeCouchbaseLite {
+			t.Logf("Don't include deleteVersion from Couchbase Lite peers when determining lastWrite %s, CBG-4432", peerName)
+			continue
+		}
+		t.Logf("deleteVersion on peer %s: %+v, implicit version: %+v", peerName, deleteVersion, deleteVersion.ImplicitHLV)
```

Comment:
Is implicit version helpful to log here? I think logging with `%#v` will log all the details. Whether or not something has an implicit hlv only indicates whether it was written by couchbase server, not via XDCR.

Reply:
Agree - will remove the redundant implicitVersion..